README.md

```diff
@@ -2422,9 +2422,13 @@ Map  entries in Hazelcast are partitioned across the cluster members. Hazelcast
 These benefits do not come for free, please consider the following trade-offs:
 
 - Members with a Near Cache will have to hold the extra cached data, which increases memory consumption.
```

Comment:
I thought that 'Clients with a Near Cache will have to hold the extra cached data, which increases memory consumption.'
could be a substitute for this item.